ci/build_docs.sh

```diff
@@ -39,7 +39,7 @@ popd
 
 if [[ "${RAPIDS_BUILD_TYPE}" == "branch" ]]; then
   rapids-logger "Upload Docs to S3"
-  aws s3 sync --delete doxygen/html "s3://rapidsai-docs/librmm/${VERSION_NUMBER}/html"
-  aws s3 sync --delete python/docs/_html "s3://rapidsai-docs/rmm/${VERSION_NUMBER}/html"
-  aws s3 sync --delete python/docs/_text "s3://rapidsai-docs/rmm/${VERSION_NUMBER}/txt"
+  aws s3 sync --only-show-errors --delete doxygen/html "s3://rapidsai-docs/librmm/${VERSION_NUMBER}/html"
+  aws s3 sync --only-show-errors --delete python/docs/_html "s3://rapidsai-docs/rmm/${VERSION_NUMBER}/html"
+  aws s3 sync --only-show-errors --delete python/docs/_text "s3://rapidsai-docs/rmm/${VERSION_NUMBER}/txt"
```

Comment:
```suggestion
  aws s3 sync --no-progress --delete doxygen/html "s3://rapidsai-docs/librmm/${VERSION_NUMBER}/html"
  aws s3 sync --no-progress --delete python/docs/_html "s3://rapidsai-docs/rmm/${VERSION_NUMBER}/html"
  aws s3 sync --no-progress --delete python/docs/_text "s3://rapidsai-docs/rmm/${VERSION_NUMBER}/txt"
```

Let's opt for `--no-progress` so that we get some output. From the S.O. article below, it looks like this option will at least give us the transfer paths which is useful as a sanity check.

- https://stackoverflow.com/a/57527597